specs.json

```diff
@@ -0,0 +1,2816 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "tifRestAPI",
+    "description": "API used for the TiF mobile app",
+    "version": "0.1"
+  },
+  "servers": [
+    {
+      "url": "{apiEndpoint}/{environment}",
+      "variables": {
+        "environment": {
+          "default": "staging",
+          "enum": [
+            "stagintTest",
+            "staging",
+            "production"
+          ]
+        },
+        "apiEndpoint": {
+          "default": "URL_GOES_HERE"
+        }
+      }
+    }
+  ],
+  "components": {
+    "schemas": {},
+    "parameters": {}
+  },
+  "paths": {
+    "/user": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Object with user data.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "format": "uuid"
+                    },
+                    "handle": {
+                      "type": "string"
+                    }
```

Comment:
I wonder if its possible to embed the regex we use for `UserHandle` in this schema itself, possibly with another zod helper function.

Reply:
I can do that by adding the regex to the zod schema